dvc/_debug.py

```diff
@@ -100,12 +122,18 @@ def debugtools(args: "Namespace" = None, **kwargs):
         if kw.get("instrument") or kw.get("instrument_open"):
             stack.enter_context(instrument(kw.get("instrument_open", False)))
         if kw.get("yappi"):
-            from datetime import datetime
-
             output = "callgrind.dvc-{0:%Y%m%d}_{0:%H%M%S}.out"
             stack.enter_context(
                 yappi_profile(path=lambda: output.format(datetime.now()))
             )
+        if kw.get("viztracer"):
```

Comment:
It's a debugging flag, so I'd like to type/use it fast. So if we need to specify depth, let's not require explicit `--viztracer` flag.
```suggestion
        if kw.get("viztracer") or kw.get("viztracer_depth"):
```